packages/bits/demo/src/components/demo/wizard-v2/wizard-restore-state/wizard-restore-state.example.component.html

```diff
@@ -23,12 +23,16 @@
                                             i18n>
                         This is required
                     </nui-validation-message>
+                </nui-form-field>
+
+                <nui-checkbox formControlName="privacy" required>
```

Comment:
it seems like this should be inside a `nui-form-field` as well?

Reply:
done